README.rst

```diff
@@ -2,107 +2,56 @@
-            /></a>
-        </td>
-        <td>
-            <a href="https://app.element.io/#/room/#sunpy:openastronomy.org"
-            ><img
-                src="https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=matrix.org"
-                alt="Matrix Chat Room"
-            /></a>
-        </td>
-        </tr>
-        <tr>
-        <td>
-            <a href="https://anaconda.org/conda-forge/sunpy"
-            ><img src="https://anaconda.org/conda-forge/sunpy/badges/version.svg" alt="Latest Conda Version"
-            /></a>
-        </td>
-        <td>
-            <a href="https://www.repostatus.org/#active"
-            ><img
-                src="https://www.repostatus.org/badges/latest/active.svg"
-                alt="Project Status: Active - The project has reached a stable, usable state and is being actively developed."
-            /></a>
-        </td>
-        <td>
-            <a href="https://community.openastronomy.org/"
-            ><img
-                src="https://cdck-file-uploads-global.s3.dualstack.us-west-2.amazonaws.com/try2/original/1X/5e1e3b3cada2d7fbae4734d4bc53999933d71c95.svg"
-                height="20px"
-                alt="OpenAstronomy Discourse community"
-            /></a>
-        </td>
-        </tr>
-        <tr>
-        <td>
-            <a href="https://zenodo.org/badge/latestdoi/2165383"
-            ><img src="https://zenodo.org/badge/2165383.svg" alt="Zenodo - Latest DOI"
-            /></a>
-        </td>
-        <td>
-            <a href="https://github.com/sunpy/sunpy/actions/workflows/ci.yml"
-            ><img
-                src="https://github.com/sunpy/sunpy/actions/workflows/ci.yml/badge.svg?branch=main"
-                alt="Continuous Integration Status"
-            /></a>
-        </td>
-        <td>
-            <a href="https://groups.google.com/g/sunpy"
-            ><img
-                src="https://fonts.gstatic.com/s/i/productlogos/groups/v9/web-48dp/logo_groups_color_1x_web_48dp.png"
-                height="20px"
-                alt="Google Groups Mailing List"
-            /></a>
-        </td>
-        </tr>
-        <tr>
-        <td>
-            <a href="https://docs.sunpy.org/"
-            ><img src="https://readthedocs.org/projects/sunpy/badge/?version=stable" alt="sunpy Stable Documentation"
-            /></a>
-        </td>
-        <td>
-            <a href="https://codecov.io/gh/sunpy/sunpy"
-            ><img src="https://codecov.io/gh/sunpy/sunpy/branch/main/graph/badge.svg" alt="CodeCov Code Coverage"
-            /></a>
-        </td>
-        <td>
-            <a href="https://numfocus.org"
-            ><img
-                src="https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A"
-                alt="Powered by NumFOCUS"
-            /></a>
-        </td>
-        </tr>
-    </tbody>
-    </table>
-    <br />
+|SunPy Logo|
```

Comment:
Modifying this is going to be a pain that we will never escape.

Reply:
The rst table doesn't look too bad? I appreciate reflowing tables is a pain. We could make it much wider in terms of columns to make it less likely we need to reflow it?

Reply:
Reflowed to much wider